src/map/packets/inventory_size.cpp

```diff
@@ -36,7 +36,7 @@ along with this program.  If not, see http://www.gnu.org/licenses/
 CInventorySizePacket::CInventorySizePacket(CCharEntity* PChar)
 {
     this->type = 0x1C;
-    this->size = 0x1A;
+    this->size = 0x64;
```

Comment:
This should use the length() setter, packet size in bytes is 100 (0x64).  If setting directly, this should be halved.